frontend/.eslintrc.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2020,
+    project: 'tsconfig.json',
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+    settings: {
+      react: {
+        version: 'detect',
+      },
+    },
+  },
+  plugins: ['@typescript-eslint/eslint-plugin'],
+  extends: [
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:cypress/recommended',
+  ],
+  root: true,
+  env: {
+    browser: true,
+    jest: true,
+    es6: true,
+  },
+  rules: {
+    '@typescript-eslint/ban-ts-comment': 'off',
+    '@typescript-eslint/interface-name-prefix': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-non-null-assertion': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/no-unused-vars': ['warn', { ignoreRestSiblings: true }],
+    '@typescript-eslint/no-empty-function': 'warn',
+    'react/display-name': 'off',
+    'react/prop-types': ['error', { skipUndeclared: true }],
+    'no-console': ['warn', { allow: ['warn', 'error'] }],
+    'react/react-in-jsx-scope': 'off',
+  },
+  overrides: [
+    {
+      files: ['**/*.tsx'],
+      rules: {
+        'react/prop-types': 'off',
+        'react-hooks/exhaustive-deps': [
+          'warn',
+          {
+            additionalHooks: 'useRecoilCallback',
```

Comment:
NO

Reply:
I'd rather deal with a warning than a hard to find bug.